crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -568,6 +565,71 @@ fn handle_declare_deploy_result<T: CairoSerialize>(
     Ok(CheatcodeHandlingResult::from_serializable(result))
 }
 
+// TODO: Should be removed once we don't hold `vm_resources_to_sierra_gas` implementation
+// Copied blockifier code
+// https://github.com/starkware-libs/sequencer/blob/5bc3098ebd04facfc3868fd09d818392748a81b4/crates/blockifier/src/bouncer.rs#L590
+fn n_steps_to_gas(n_steps: usize, versioned_constants: &VersionedConstants) -> GasAmount {
+    let gas_per_step = versioned_constants
+        .os_constants
+        .gas_costs
+        .base
+        .step_gas_cost;
+    vm_resource_to_gas_amount(n_steps, gas_per_step, "steps")
+}
+
+// TODO: Should be removed once we don't hold `vm_resources_to_sierra_gas` implementation
+// Copied blockifier code
+// https://github.com/starkware-libs/sequencer/blob/5bc3098ebd04facfc3868fd09d818392748a81b4/crates/blockifier/src/bouncer.rs#L578C1-L589C1
+fn vm_resource_to_gas_amount(amount: usize, gas_per_unit: u64, name: &str) -> GasAmount {
+    let amount_u64 = u64_from_usize(amount);
+    let gas = amount_u64.checked_mul(gas_per_unit).unwrap_or_else(|| {
+        panic!(
+            "Multiplication overflow converting {name} to gas. units: {amount_u64}, gas per unit: {gas_per_unit}.",
+        )
+    });
+
+    GasAmount(gas)
+}
+
+// TODO: Should be removed once we don't hold `vm_resources_to_sierra_gas` implementation
+// Copied blockifier code
+// https://github.com/starkware-libs/sequencer/blob/5bc3098ebd04facfc3868fd09d818392748a81b4/crates/blockifier/src/bouncer.rs#L595C4-L595C23
+fn memory_holes_to_gas(
+    n_memory_holes: usize,
+    versioned_constants: &VersionedConstants,
+) -> GasAmount {
+    let gas_per_memory_hole = versioned_constants
+        .os_constants
+        .gas_costs
+        .base
+        .memory_hole_gas_cost;
+    vm_resource_to_gas_amount(n_memory_holes, gas_per_memory_hole, "memory_holes")
+}
+
+// TODO: Should be taken from blockifier
+// Copied blockifier code
+// https://github.com/starkware-libs/sequencer/blob/5bc3098ebd04facfc3868fd09d818392748a81b4/crates/blockifier/src/bouncer.rs#L649
+fn vm_resources_to_sierra_gas(
```

Comment:
Why do we have to copy it? It seems to be publish now.

Reply:
On [`main-v0.14.0`](https://github.com/starkware-libs/sequencer/tree/main-v0.14.0) it's not present yet.